### PR TITLE
Added screens

### PR DESCRIPTION
**Pull request by param-patel-08**
Added Favourites screen
Added Visited screen
Added Bookmarks screen
In UserProfileView,
added on-click event handlers for 3 screens
added navigation 
updated _layout.tsx

### CODE REVIEW

**Review by param-patel-08 — COMMENTED** [review]
There are some conflicts will look into it later.
Working on the frontend part of the screens now

File: app/(tabs)/ProfileView/BookmarkedSpotsView.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, FlatList, Image } from 'react-native';
+import { AntDesign } from '@expo/vector-icons';
+import { NavigationProp, useNavigation } from '@react-navigation/native';
+
+interface BookmarkedSpot {
```
Comment (Joshua-Frost) [inline app/(tabs)/ProfileView/BookmarkedSpotsView.tsx:6]: Check out the useContext() function when you are finished to import the bookmarkedRestaurants variable that holds a list of Restaurant objects (or will do when the API calls are done)

File: app/(tabs)/ProfileView/FavoriteSpotsView.tsx [hunk 2]
```diff
@@ -11,9 +12,10 @@ interface FavouriteSpot {
 const favouriteSpots: FavouriteSpot[] = [
```
Comment (Joshua-Frost) [inline app/(tabs)/ProfileView/FavoriteSpotsView.tsx:12]: Same with this one
const { favouriteRestaurants } = useContext(App context);
For the real data (once the API calls are implemented)

File: dwad [hunk 3]
```diff
@@ -0,0 +1,176 @@
+* [33m5364b29[m[33m ([m[1;36mHEAD -> [m[1;32mmain[m[33m, [m[1;31morigin/param[m[33m, [m[1;32mparam[m[33m)[m Updated Profile Section Updated the profile icon Added arrows and icons with colors. updated the css design
```
Comment (Joshua-Frost) [inline dwad]: I wonder what this file is?

**Review by Joshua-Frost — APPROVED** [review]
All fixed

**Pull request merged**